### PR TITLE
Adding disk size as a parameter to AWS instance create

### PR DESCRIPTION
**Pull request by bretttegartms**

### CODE REVIEW

File: lib/python/treadmill_aws/cli/admin/aws/instance.py [hunk 1]
```diff
@@ -96,8 +96,16 @@ def configure(instance):
         type=int,
         help='Number of instances'
     )
+    @click.option(
+        '--disk',
+        required=True,
+        default=10,
+        type=int,
```
Comment (andreikeis) [inline lib/python/treadmill_aws/cli/admin/aws/instance.py]: put a todo to convert it to qualified string. style failed anyways, so you need to make a change.

Comment (bretttegartms) [reply lib/python/treadmill_aws/cli/admin/aws/instance.py]: Why does it need to be converted to string? boto3 expects an integer value.

Comment (andreikeis) [reply lib/python/treadmill_aws/cli/admin/aws/instance.py]: i find --size 10 confusing from ux perspective. --disk-size 10GB is easier to read. so - not important, not now if ever.

Comment (bretttegartms) [reply lib/python/treadmill_aws/cli/admin/aws/instance.py]: That makes sense- I changed the CLI option and added a callback to convert it to int.

**Pull request merged**